app/cmd/lintAndInstall.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2018 The Helm Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/helm/chart-testing/pkg/chart"
+	"github.com/helm/chart-testing/pkg/config"
+
+	"github.com/spf13/cobra"
+)
+
+func newLintAndInstallCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lint-and-install",
```

Comment:
A short use command would be nice, too. Maybe something like `li`.

Reply:
Fixed.